akka-persistence/src/main/protobuf/MessageFormats.proto

```diff
@@ -45,3 +45,9 @@ message PersistentStateChangeEvent {
   required string stateIdentifier = 1;
   optional string timeout = 2;
 }
+
+message PersistentFSMSnapshot {
+  required string stateIdentifier = 1;
+  required PersistentPayload data = 2;
+  optional string timeout = 3;
```

Comment:
@patriknw I've used ```string``` in ```PersistentStateChangeEvent```, so I've just done the same for ```PersistentFSMSnapshot```. I can use ```int32``` in ```PersistentFSMSnapshot```, it's probably not worth it to change ```PersistentStateChangeEvent```, since it will break recovery, right?